### PR TITLE
Email templates

### PR DESCRIPTION
**Pull request by Irev-Dev**
In this branch the pug email templates were set up,
but then parcel was re done to output multiple files and pug templates were enabled for the rest of the web-server as well.

### CODE REVIEW

File: server/controllers/userController.js [hunk 1]
```diff
@@ -27,15 +28,22 @@ exports.setToken = async (req, res, next) => {
...
     resetURL,
-    // filename: 'password-reset', // wesbos's file name used for PUG templates
+    subscribeURL,
+    filename: 'Verify', // wesbos's file name used for PUG templates
   });
-  next();
+  //next();
+  req.flash('success', `you did it!! ${req.body.name}`);
+  // res.render('main', {});
+  res.redirect('/wtf');
```
Comment (mrhut10) [inline server/controllers/userController.js:46]: whats the story with the redirect to /wtf???
@Irev-Dev

File: server/controllers/userController.js [hunk 2]
```diff
@@ -48,3 +56,16 @@ exports.verifyToken = async (req, res, next) => {
...
+  if(!user) {
+    return res.send('error');
+  }
+  user.subscribeSetting = 'unsubscribed';
+  user.subscribeUpdateDate = Date.now();
+  user.subscribeUpdateFrom = req.query.date;
+  user.subscribeUpdateSource = req.query.source;
+  await user.save();
+  res.send('sorry to se you go :(');
+};
```
Comment (mrhut10) [inline server/controllers/userController.js:71]: are we handling the case where someone resigns up??
@Irev-Dev

**Review by mrhut10 — APPROVED** [review]
looks great @Irev-Dev 
just wondering if we are covering the case of someone resubscribing via the form or another method.

**Comment by mrhut10** [comment]
email templates, html templates and flashes working

**Comment by mrhut10** [comment]
i would delete the branch however i believe worthwhile having a email template branch in the future.

File: server/controllers/userController.js (hunk 2 above)
Comment (Irev-Dev) [reply server/controllers/userController.js:71]: not at this point no :/

Comment (mrhut10) [reply server/controllers/userController.js:71]: Ok, maybe we send them a welcome back email.

**Pull request merged**